solr_conf/conf/managed-schema

```diff
@@ -287,6 +299,9 @@
   <copyField source="tags_ss_lower" dest="tags_t" maxChars="30000" />
   <copyField source="type_s" dest="type_t" maxChars="300" />
 
+  <!-- copy description to a nostem field for exact matches -->
+  <copyField source="description_en_bigram" dest="description_nostem" maxChars="30000" />
```

Comment:
what is the default value for`stored` ? maybe good to explicitly set stored to false

Reply:
Does this need to be set separately for the `copyField` call and the `dynamicField` that it matches?

Reply:
I don't think so, I think it's defined on the dynamic field type

Reply:
I thought so as well! In that case, already done 😄